contracts/StablecoinConverter.sol

```diff
@@ -386,11 +518,21 @@ contract StablecoinConverter is EpochTokenLocker {
         );
         return (executedBuyAmount, executedSellAmount);
     }
+    // Private pure
 
+    /** @dev used to determine if an order is valid for specific auction/batch
+      * @param order object whose validity is in question
+      * @param batchIndex auction index of validity
+      * @return true if order is valid in auction batchIndex else false
+      */
     function checkOrderValidity(Order memory order, uint batchIndex) private pure returns (bool) {
         return order.validFrom <= batchIndex && order.validUntil >= batchIndex;
     }
 
+    /** @dev Token ordering is verified by submitSolution. Required because binary search is used to fetch token info.
+      * @param tokenIdsForPrice list of tokenIds
+      * @return true if tokenIdsForPrice is sorted else false
+      */
     function checkPriceOrdering(uint16[] memory tokenIdsForPrice) private pure returns (bool) {
```

Comment:
unrelated: Can't this function go into the tokenConservation library?

Reply:
Ya Probably. Should I do it after this is ready?